packages/runtime/container-runtime/src/connectionTelemetry.ts

```diff
@@ -161,7 +198,7 @@ class OpPerfTelemetry {
                 referenceSequenceNumber: message.referenceSequenceNumber,
                 duration,
                 category,
-                pingLatency: this.pingLatency,
+                pingLatency: this.pingLatency
```

Comment:
you will get eslint violation - comma is required here.

Reply:
yep, my bad.